changelog/15044.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
Comparing against the 1.17.7 PR @mladlow had mentioned this should be within the changes section, not improvements. Looks like the the PR kept it as improvement but the release notes 

https://github.com/hashicorp/vault/pull/14232/files#r817105897